ReadMe.md

```diff
@@ -33,7 +33,7 @@ Here's a list of all the features of our GitHub Profile ReadMe customisation too
 * ⏰ Saves time by providing all available components and customisation options in one place.
 * 💪🏻 Helps developers create high-quality and informative GitHub Profile ReadMe pages that showcase their skills and experience.
 
-With our GitHub profile ReadMe customisation tool, you can take your GitHub profile to the next level and impress potential employers and collaborators. Try it out today and see the difference it can make! 🤗
+With our GitHub profile README customisation tool, you can take your GitHub profile to the next level and impress potential employers and collaborators. Try it out today and see the difference it can make! 🤗
```

Comment:
Make this all as "GitHub Profile ReadMe" - It's the product name! 💪

Reply:
Okay